std_msgs/Header header
uint32 cluster_iterations
uint32 total_iterations
sensor_msgs/PointCloud2 cloud
